index.html

```diff
@@ -198,6 +205,10 @@ <h1>The Registration Process</h1>
       <li>
 Properties in the DID Core Registries MUST NOT be removed, only deprecated.
       </li>
+      <li>
+Properties definitions in the DID Core Registries MUST include a Concise Data Definition Language (CDDL) [[RFC8610]] of the 
+Property and it's Abstract Data model structure. 
```

Comment:
This needs to make it clear that CDDL definitions must be done on a per-representation/per-MIME type basis.

Reply:
see issue to track this conversation: https://github.com/w3c/did-spec-registries/issues/153